ui/raidboss/data/07-dt/raid/r8s.ts

```diff
@@ -1,14 +1,1907 @@
+        data.hasSpread = undefined;
+        data.isFirstRage = false;
+      },
+      outputStrings: {
+        spreadThenStack: Outputs.spreadThenStack,
+        stackThenSpread: {
+          en: '${stack} => Spread',
+        },
+        spread: Outputs.spread,
+        stackOnPlayer: Outputs.stackOnPlayer,
+        stackOnYou: Outputs.stackOnYou,
+      },
+    },
+    {
+      id: 'R8S Beckon Moonlight Quadrant Two',
+      type: 'StartsUsing',
+      // A3C2 => Moonbeam's Bite dash with Left cleave
+      // A3C3 => Moonbeam's Bite dash with Right cleave
+      netRegex: { id: ['A3C2', 'A3C3'], source: 'Moonlit Shadow', capture: true },
+      condition: (data) => {
+        data.moonbeamBitesTracker = data.moonbeamBitesTracker + 1;
+        if (data.moonbeamBitesTracker === 2)
+          return true;
+        return false;
+      },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime),
+      infoText: (data, _matches, output) => {
+        return output.safeQuad!({ quad: data.moonlightQuadrant2 });
+      },
+      outputStrings: moonlightOutputStrings,
+    },
+    {
+      id: 'R8S Weal of Stone Cardinals',
+      // There are two casts and cardinals is always safe:
+      // A791 Weal of Stone
+      // A792 Weal of Stone
+      // Due to timing of this impacting Windfang/Stonefang, call earlier
+      // Using Moonbeam Bites +1s for spread (A3BF) and stack (A3C0) abilities to complete
+      type: 'StartsUsing',
+      netRegex: { id: ['A3C2', 'A3C3'], source: 'Moonlit Shadow', capture: true },
+      condition: (data) => {
+        if (data.moonbeamBitesTracker === 4)
+          return true;
+        return false;
+      },
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) + 1,
+      infoText: (_data, _matches, output) => {
+        return output.cardinals!();
+      },
+      outputStrings: {
+        cardinals: Outputs.cardinals,
+      },
+    },
+    // Phase 2
+    {
+      id: 'R8S Quake III',
+      type: 'StartsUsing',
+      netRegex: { id: 'A45A', source: 'Howling Blade', capture: false },
+      alertText: (_data, _matches, output) => output.healerGroups!(),
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      // headmarkers with casts:
+      // A45D (Ultraviolent Ray)
+      // TODO: Determine platform to move to based on player positions/role?
+      id: 'R8S Ultraviolent Ray Target',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.ultraviolent },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => {
+        return output.uvRayOnYou!();
+      },
+      run: (data) => data.hasUVRay = true,
+      outputStrings: {
+        uvRayOnYou: {
+          en: 'UV Ray on YOU',
+        },
+      },
+    },
+    {
+      // Gleaming Fang's cast Gleaming Beam (A45E) 2.1s after ActorControlExtra
+      // This ActorControlExtra is unique to the Ultraviolent Ray and Champion's Circuit
```

Comment:
Just a nitpick:
- `category: '0197'` is `PlayActionTimeline`
- `param1: '11D3'` is the actual action timeline to play

I would suggest re-wording to something like `PlayActionTimeline param1 of '11D3' is unique...`

Reply:
Thanks for the clarification! I've made the fix.